library/core/src/main/java/com/google/android/exoplayer2/upstream/DefaultDataSource.java

```diff
@@ -60,6 +61,7 @@
   private DataSource contentDataSource;
   private DataSource rtmpDataSource;
   private DataSource dataSchemeDataSource;
+  private DataSource rawResourceDataSource;
```

Comment:
Please update the list in the Javadoc at the top of this file to include an entry:

<li>rawresource: For fetching data from a raw resource in the applications' apk (e.g. rawresource:///resourceId, where rawResourceId is the integer identifier of the raw resource).</li>

Probably inserted under the asset one, since they seem related.